frog/imports/client/Wiki/components/TopNavbar/index.js

```diff
@@ -7,8 +7,9 @@ import OverflowPanel from './OverflowPanel';
 
 type TopNavBarPropsT = {
   /** The current Meteor username */
```

Comment:
this kind of comment is not very useful. Try to limit comments to things that are not obvious.

Reply:
This comment is not a part of the PR.

Reply:
Removed the comment